omod/src/main/java/org/openmrs/module/spa/utils/SpaModuleUtils.java

```diff
@@ -49,15 +49,24 @@ public static File getSpaStaticFilesDir () {
         return folder;
     }
 
+    public static boolean isRemoteAssetsEnabled() {
+        AdministrationService administrationService = Context.getAdministrationService();
+        String isRemoteAssetEnabled = administrationService.getGlobalProperty(GP_IS_REMOTE_ENABLED, "false");
+        log.info("Remote/Local frontend assets enabled {}", isRemoteAssetEnabled);
+        return Boolean.parseBoolean(isRemoteAssetEnabled);
+    }
+
     /*
      * This should take care for File system, Class path, and URL path
      */
-    public static String getFrontendDirectoryPath() {
+    public static String getRemoteAssetsUrl() {
         AdministrationService administrationService = Context.getAdministrationService();
-        String path = administrationService.getGlobalProperty(SpaConstants.FRONTEND_DIRECTORY_PATH, "/frontend");
+        //Defaults to SPA environment at least for now.
```

Comment:
```suggestion
        // FIXME Defaults to SPA environment at least for now.
```

Reply:
I actually don't know what this comment means. Do you? Or @corneliouzbett ?

Reply:
I took it to be trying to explain why the default value is `https://spa-modules.nyc3.digitaloceanspaces.com/@openmrs/esm-app-shell/latest/` (is that actually a useful remote URL?)